mysql-test/suite/encryption/t/tablespace_encryption.test

```diff
@@ -0,0 +1,161 @@
+--disable_warnings
```

Comment:
It's the default, just remove the i_f_p_t=ON

Reply:
done